MiniSim/Service/DeviceService.swift

```diff
@@ -359,7 +441,19 @@ extension DeviceService {
 
   static func getAndroidDevices() throws -> [Device] {
     Thread.assertBackgroundThread()
+    let emulators = try getAndroidEmulators()
+    let devices = try getAndroidPhysicalDevices()
```

Comment:
Same here, if physical devices fail we want to display emulators

Reply:
Check my other comment to make sure it is what you want

Reply:
done